_stbt/core.py

```diff
@@ -1259,7 +1259,7 @@ class NoValue(object):
         stable_duration = t - stable_since
 
         if value and stable_duration >= stable_secs:
```

Comment:
This could be `if value and t - stable_since >= stable_secs`.  That way `stable_since` can be `None` (or even not set) until `value`.